vmdb/lib/workers/schedule_worker.rb

```diff
@@ -233,6 +216,23 @@ def schedules_for_scheduler_role
       @queue.enq :storage_scan_timer
     end
 
+    schedules_for_ems_refresh
+  end
+
+  def schedules_for_ems_refresh
+    config = VMDB::Config.new("vmdb").config.fetch(:ems_refresh, {})
+
+    ExtManagementSystem.leaf_subclasses.each do |klass|
```

Comment:
All leaves are concrete.  Otherwise, I'd have to enumerate every class which defeats the point of figuring it out.  I can change the name if you want, but not in this PR.  PRs are welcome :trollface: